Week3/week3HomeWork/Exercise3/fortuneTeller.js

```diff
@@ -0,0 +1,39 @@
+const numChildren = [1, 2, 3, 4, 5];
+const partnerNames = ["Jessica", "Wilma", "Nadia", "Susan", "Angelina"];
+const locations = ["Syria", "London", "Turkey", "Netherland", "Franca"];
+const jobs = ["accountant", "brogrammer", "salesman", "businessman", "manager"];
+
+function tellFortune(child, partener, location, job) {
+    const numChildren = [1, 2, 3, 4, 5];
+    const partnerNames = ["Jessica", "Wilma", "Nadia", "Susan", "Angelina"];
+    const locations = ["Syria", "London", "Turkey", "Netherland", "Franca"];
+    const jobs = [
+        "accountant",
+        "brogrammer",
+        "salesman",
+        "businessman",
+        "manager"
+    ];
+    child = numChildren[Math.floor(Math.random() * 5)];
```

Comment:
Here I have the same question as before, this works. But what if there are more (or less) than 5 children in the array?

Reply:
then it give error . i should write it array.length-1